packages/hoppscotch-app/src/components/collections/index.vue

```diff
@@ -44,6 +44,13 @@
           @click="displayModalAdd(true)"
         />
         <span class="flex">
+          <ButtonSecondary
+            v-tippy="{ theme: 'tooltip' }"
+            blank
+            :title="Order"
```

Comment:
There should be an `i18n` tag here.
Also this bound variable doesn't exist.
`:prop`'s assigned value is not considered as a string, but as a JS expression.

React equivalent
```
<ButtonSecondary title={{ Order }} />
```